pages/api/study/[key].js

```diff
@@ -0,0 +1,47 @@
+import { useCors } from '@lib/cors'
+import dbConnect from '@lib/db-connect'
+import ParticipantData from '@models/participantData'
+import Study from '@models/study'
+
+export default async function handler(req, res) {

```

Comment:
Should probably separate this into 2 files. 
- /api/study/validate/[key].js for GET request from mobile app
or even /api/study/metadata/[key].js if we want to avoid verb in path
- /api/study/data/index.js for POST request and specify key in request body instead of in query param